libethereum/BlockQueue.cpp

```diff
@@ -237,14 +217,12 @@ ImportResult BlockQueue::import(bytesConstRef _block, bool _isOurs)
 	// Check it's not in the future
 	if (bi.timestamp() > utcTime() && !_isOurs)
 	{
-		m_future.insert(make_pair((unsigned)bi.timestamp(), make_pair(h, _block.toBytes())));
+		m_future.insert((unsigned)bi.timestamp(), h, _block.toBytes());
 		char buf[24];
 		time_t bit = (unsigned)bi.timestamp();
```

Comment:
I'm not sure using `unsigned` in place of `time_t` is good idea. Maybe use `time_t` directly in `m_future`?